sdk/storage/storage-file-share/src/models.ts

```diff
@@ -189,6 +191,16 @@ export interface ShareProtocols {
   nfsEnabled?: boolean;
 }
 
+export interface ShareClientConfig {
+  /** Valid value is backup */
+  fileRequestIntent?: ShareFileRequestIntent;
```

Comment:
The documentation string needs to explain this more completely. Is this saying "backup" is the _ONLY_ valid value? Why is "none" presented as an option in the `ShareFileRequestIntent`?

Reply:
Swagger spec has been updated. It's defined as an extendable enum now. Currently there's only one valid value, but should expect more values in the future.